packages/builders/src/interactions/slashCommands/mixins/ApplicationCommandOptionWithChoicesAndAutocompleteMixin.ts

```diff
@@ -4,9 +4,15 @@ import { validateMaxChoicesLength } from '../Assertions';
 
 const stringPredicate = z.string().min(1).max(100);
 const numberPredicate = z.number().gt(-Infinity).lt(Infinity);
-const choicesPredicate = z.tuple([stringPredicate, z.union([stringPredicate, numberPredicate])]).array();
+const choicesPredicate = z
+	.object({ name: stringPredicate, value: z.union([stringPredicate, numberPredicate]) })
+	.array();
 const booleanPredicate = z.boolean();
 
+export interface ChoiceData<T extends number | string> {
+	name: string;
+	value: T;
+}
```

Comment:
smallest of nits, but this should have an empty line after it :p

Reply:
done!